### DISCUSSION TITLE
Docker Compose Settings

### DISCUSSION THREAD

**Discussion by sherman94062** [implydata/learn-druid#55] [Q&A] [answered]
Hey all, love, love, **LOVE** this repo!!  When I last ran Druid in Docker, I ran into issues with memory and disk space.  I believe that when you bring up the Docker cluster, you need to specify the following in your docker-compose.yml:
volumes:
  metadata_data: {}
  middle_var: {}
  historical_var: {}
  broker_var: {}
  coordinator_var: {}
  router_var: {}
  druid_shared: {}

Did you all set those up so that this cluster can "persist" if we need to bring it down?

**Comment by techdocsmith** [comment, marked as answer]
Thanks for the question @sherman94062 , cross posting the answer from slack, here.

> The docker volumes are created if they do not exist and will persist through a shutdown or restart of the services. In order to remove them, you need to use the -v flag; the default (no -v flag) will keep the volumes available for the next restart.
